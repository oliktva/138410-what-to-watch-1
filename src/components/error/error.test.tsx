import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

import Error from './error';

Enzyme.configure({adapter: new Adapter()});

it(`renders correctly`, () => {
  const error = shallow(<Error />);

  expect(toJson(error)).toMatchSnapshot();
});
