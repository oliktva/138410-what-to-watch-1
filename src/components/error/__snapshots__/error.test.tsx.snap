// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<PageWrapper>
  <div
    className="user-page"
  >
    <Header
      className=""
      isLightLogo={true}
      withBreadcrumbs={false}
    />
    <div
      className="page-content"
    >
      <h1>
        Internal server error
      </h1>
      <p>
        Please try again later
      </p>
      <Footer />
    </div>
  </div>
</PageWrapper>
`;
